server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -284,6 +284,18 @@ public class ShuffleServerConf extends RssBaseConf {
       .defaultValue(0L)
       .withDescription("For localstorage, it will exit when the failed initialized local storage exceed the number");
 
+  public static final ConfigOption<Boolean> BUFFER_FLUSH_ENABLED = ConfigOptions
+       .key("rss.server.buffer.flush.enabled")
```

Comment:
Could we change the name from `rss.server.buffer.flush.enabled` to `rss.server.single.buffer.flush.enabled`?

Reply:
done